ee/tasks/subscriptions/__init__.py

```diff
@@ -67,10 +67,12 @@ def _deliver_subscription_report(
         except Exception as e:
             incr("subscription_slack_send_failure")
             logger.error(e)
+    elif subscription.target_type == "in_app_notification":
+        incr("subscription_in_app_notification_success")
     else:
         raise NotImplementedError(f"{subscription.target_type} is not supported")
 
-    if not is_new_subscription_target:
+    if not is_new_subscription_target and subscription.target_type != "in_app_notification":
```

Comment:
Is this the right thing to compare to? Somehow I'd `and subscription.frequency != 'on_change'` feels more correct for checking to set the next date or not...?